pom.xml

```diff
@@ -1291,8 +1291,8 @@
                         </image>
                         <image>
                             <repository>${dockerHubPublic}/cafapi/opensuse-python3</repository>
-                            <tag>4.0.1</tag>
-                            <digest>sha256:18628bccbf6c15f3335e1e62e5e786a6bce55ddbd244e6bfa79d0c6f09790e28</digest>
+                            <tag>2.8.1</tag>
+                            <digest>sha256:9e904b4c1f732688e4e500fccf60ff1f25ef629dcf3b0a1f274fa3af8c026703</digest>
```

Comment:
Why does this appear to be downgrading?

Reply:
The 'latest' tag has not been set on the newest release
https://github.com/CAFapi/opensuse-base-images/releases/tag/v4.0.1

Reply:
That's interesting.  We'll have to look into what's going on there.  For now I'm manually updated the 'latest' tag.